frontend/court-up/src/pages/Home/Home.css

```diff
@@ -0,0 +1,51 @@
+:root{
+    --accent-1: #FF5814;
+    --accent-2: #565656;
+    --text-1: #FFFFFF;
+    --background: #1B1B1B;
+
+}
```

Comment:
im being picky. u could put spaces between classes to satisfy my pickiness

Reply:
no